PayPal/src/main/java/com/braintreepayments/api/PayPalRequest.java

```diff
@@ -281,6 +296,10 @@ public boolean hasUserLocationConsent() {
         return hasUserLocationConsent;
     }
 
+    public boolean isUseAppLinkReturn() {
```

Comment:
Thoughts on renaming the variable to make the getter more readable? Maybe `appLinkEnabled` (`setAppLinkEnabled()` and `isAppLinkEnabled`)?

Reply:
+1 on `appLinkEnabled` naming is more clear

Reply:
quick question: Could we infer this value internally by checking if `BraintreeClient` is configured to use app links? 

e.g.
```kotlin
val hasAppLinkReturnUri = (braintreeClient.appLinkReturnUri != null)
```

Reply:
Hmm that's a good call. If a merchant were to set a URI on the BraintreeClient, they'd probably want to use App Links. Is there a benefit in having them opt-in at the request level?

Reply:
My concern would be that since we are adding app link support only for PayPal currently, when we add support for future payment methods, merchants would be defaulted into using app links if they had already set them on their BraintreeClient (for PayPal specifically), and they might not be set up to use app links for the other payment flows (i.e. maybe they want to return to a different activity etc)

Reply:
In those cases would merchants be creating a new `BraintreeClient` instance where they would omit the app link URI?

Reply:
Yes they could, so there would be a work around - I guess my concern would be merchants that have a single `BraintreeClient`, if they set up for app links for PayPal, and in a future release we add app link support for another feature, then without any changes just upgrading to the latest version, the merchant would be opted into app links (which might be a breaking change for their integration)

Reply:
That makes sense. I'm good with keeping the enabled boolean in each of the request objects. These would most likely go away in v5 since we can just check if the URI is set on each of the request objects.

Reply:
Yeah it definitely makes sense in v4 to have two-step enablement for app links. We should consider moving `returnUrlScheme` and `appLinkReturnUri` properties into the request objects in v5 to make it a one-step integration in the future. A single source of truth for enabling app links should make the integration less error prone.